content/blog/pulumi-release-notes-114/index.md

```diff
@@ -0,0 +1,117 @@
+- **[Pulumi EKS 3.0 Release](/blog/eks-v3-release/)**  
+  Update introducing support for Amazon Linux 2023, Access Entries for IAM integration, EKS managed addons, and enhanced security features.
+- **[Pulumi Kubernetes Await Logic](/blog/improved-kubernetes-await-logic/)**
+  Better resource dependency tracking, improved error reporting, faster deletions with propagation policies, and custom readiness criteria support.
+
+### Provider Updates
+
+- **[Use Any Terraform Provider](/blog/any-terraform-provider/)**
+  Automatic generation of Pulumi providers from any Terraform provider, dramatically expanding the ecosystem of available resources.
+- **[AWS CDK On Pulumi 1.0](/blog/aws-cdk-on-pulumi-1.0/)**
+  Expanded compatibility with AWS CDK features and construct hub integration, allowing developers to continue using familiar AWS CDK patterns within Pulumi.
+- **[Azure v6 Release](/blog/azure-v6-release/)**  
+  Major update ensuring compatibility with the latest Azure services, improved error messages, and new resource types.
+- **[Google Cloud Provider v8.0.0](/blog/gcp-v8-release/)**
+  Comprehensive support for new Google Cloud services and improved resource management capabilities.
+- **[AWS Cloud Control Provider GA](/blog/pulumi-aws-cloudcontrol-provider/)**
+  Native AWS API support with day-zero coverage for new AWS features and improved consistency in resource management.
+
+### Core Platform Features
+
+- **[Pulumi Visual Studio Code Extension](/blog/pulumi-vscode-extension/)**  
+  Enhanced IaC debugging in the IDE, Pulumi YAML support, and ESC management within VS Code.
+- **[Pulumi Docker Image Improvements](/blog/docker-containers/)**  
+  Versioned images with pre-installed tools for Python, Node.js, and .NET. Pulumi Deployments now supports setting the version of the Node.js and Python runtimes used in the Deployment environment.
+- **[Python UV Support](blog/python-uv-toolchain/)**  
+  Built-in support for uv, an extremely fast Python package manager that speeds up dependency installation by up to 100x compared to traditional tools.
+- **DependsOn for Provider Functions**  
+  Manage explicit dependencies between provider functions for more accurate execution in complex deployments.
+
+---
+
+## Pulumi Environments, Secrets and Configuration (Pulumi ESC)
+
+### General Availability & Key Enhancements
+
+- **[ESC General Availability](/blog/pulumi-esc-ga/)**  
+  GA of Pulumi ESC -redefining secrets management with features like dynamic credentials, hierarchical environments, a secrets broker with external stores like AWS Secrets Manager, and Kubernetes operators for runtime integration.
+- **[ESC Projects & Environment Tags](/blog/esc-projects-environment-tags-launch/)**  
+  Organize secrets and configurations at scale with new grouping capabilities.
+- **[ESC Webhooks](/blog/esc-webhooks-launch/)**  
```

Comment:
Can we add Parameter store and Imported By blogs

Reply:
done!